From 913ec3477f4eb81327cd201331141f992aa3cdfd Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Wed, 12 Sep 2018 14:57:22 +0200
Subject: [PATCH] dnf: trivial: Add missing include

Add missing dnf-db.h for dnf_db_ensure_origin_pkglist() and
dnf_db_ensure_origin_pkg().
---
 backends/dnf/pk-backend-dnf.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/backends/dnf/pk-backend-dnf.c b/backends/dnf/pk-backend-dnf.c
index b6d575737..e2e4a0a7b 100644
--- a/backends/dnf/pk-backend-dnf.c
+++ b/backends/dnf/pk-backend-dnf.c
@@ -36,6 +36,7 @@
 #include <libdnf/libdnf.h>
 #include <libdnf/dnf-advisory.h>
 #include <libdnf/dnf-advisoryref.h>
+#include <libdnf/dnf-db.h>
 #include <libdnf/hy-packageset.h>
 #include <libdnf/hy-query.h>
 #include <libdnf/dnf-version.h>
-- 
2.19.0

